package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@paulrberg/solidity-template",
+  "name": "@coinvise/solidity-template",
   "description": "Template for writing Solidity smart contracts",
   "version": "1.0.0",
   "author": {
-    "name": "Paul Razvan Berg",
-    "url": "https://paulrberg.com"
+    "name": "Sebastian T F",
```

Comment:
Can you use "Coinvise" here and the organisation value in the url?

Reply:
Done: https://github.com/coinvise/solidity-template/pull/1/commits/6d7485a60584ace4f491317ea557efd73ad647bc